clap_derive/src/parse.rs

```diff
@@ -176,7 +173,13 @@ impl Parse for ClapAttr {
                 "external_subcommand" => Ok(ExternalSubcommand(name)),
                 "verbatim_doc_comment" => Ok(VerbatimDocComment(name)),
 
-                "default_value" => Ok(DefaultValue(name, None)),
+                "default_value" => {
+                    abort!(name,
+                        "`#[clap(default_value)` attribute (without a value) has been replaced by `#[clap(default_value_t)]`.";
+                        help = "Change the attribute to `#[clap(default_value_t)]`";
+                    )
```

Comment:
`cargo-up` should be able to handle this easily without us needing the abort. This should be deleted when I finish the structopt to clap upgrader then.

Reply:
I personally would find this useful because I'm unlikely to download a special tool just to handle a library migration.  The likelihood of downloading is proportionate to the number of CLIs I maintain.